### ISSUE TITLE
Is it possible to temporarily stop web-server to free up space? 

### ISSUE THREAD

**Issue by shahabmusic** [jeelabs/esp-link#317]
I have read everywhere that esp-link does not support TLS for MQTT because there is not enough space.
I have also seen other people connecting to MQTT with TLS however without webserver running on their esp8266.
Now my question is that is it possible to turn-off the webserver and free up some space so TLS can be used? TLS can be stopped and webserver can be turned back on by reseting esp8266 etc if needed.

**Comment by tve** [comment]
Yes, you could turn off listening sockets but I'm not sure how much memory you really release. There is also a lot of static memory allocated across the board. So YMMV...
